src/renderable.jl

```diff
@@ -151,5 +151,52 @@ function add_renderable!(
         va = VelocityArrow(entity=entity, color=color)
         add_renderable!(rendermodel, va)
     end
-    rendermodel
+    return rendermodel
 end
+
+
+"""
+Convenience function for rendering a scene. Takes care of initializing a `RenderModel` and updating the camera.
+For full control, use `render!(rendermodel, renderables)` instead.
+"""
+function render(
+    # TODO: what about the roadway?
+    #  A) only allow rendering with roadway (as a first or second argument), scene without a roadway is not needed
+    #  B) implement a second, separate method?
+    #  C) allow roadway to be Union{Nothing, Roadway}?
+    #  D) more options??
```

Comment:
roadway should be rendered before (i.e. below) scene. which of the above options makes most sense?

Reply:
I don't think the default should be specific to roadway (see my other comment)